packages/babel-core/register.js

```diff
@@ -1,3 +0,0 @@
-/* eslint max-len: 0 */
-// TODO: eventually deprecate this console.trace("use the `babel-register` package instead of `babel-core/register`");
-module.exports = require("babel-register");
```

Comment:
I guess the place that would be the hardest to migrate from this would be build systems; or you get into a situation where you need a build system to build a build system so you can build (your build system)...